_posts/2025-05-03-modulo-contactos.md

```
@@ -0,0 +1,41 @@
+    Solicitud,
+  ]
+comments: false
+author: Catalina Pinilla
+---
+
+## Información de contacto
+
+En `Consultas por rut` que se encuentra en la `Matriz de riesgo` podrás acceder al detalle `Ver contactos` donde podrás acceder a la información del rut que estás consultando pudiendo acceder a correos electrónicos, números de teléfono y direcciones y podrás ver la fuente de donde vienen estos datos.
+Para que sea más fácil su uso, podrás copiar estos datos para usarlos como prefieras, para hacer una llamada, copiarlos en un correo, enviar los datos en un chat, etc.
+
+#### A continuación te enseñamos como se ve
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)